bbw.ipynb

```diff
@@ -107,70 +117,69 @@
     {
      "data": {
       "text/html": [
-       "<div>\n",
-       "<style scoped>\n",
-       "    .dataframe tbody tr th:only-of-type {\n",
-       "        vertical-align: middle;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe tbody tr th {\n",
-       "        vertical-align: top;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe thead th {\n",
-       "        text-align: right;\n",
-       "    }\n",
-       "</style>\n",
        "<table border=\"1\" class=\"dataframe\">\n",
        "  <thead>\n",
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
-       "      <th>file</th>\n",
-       "      <th>column0</th>\n",
-       "      <th>column</th>\n",
-       "      <th>property</th>\n",
+       "      <th>col0</th>\n",
+       "      <th>col1</th>\n",
+       "      <th>col2</th>\n",
+       "      <th>col3</th>\n",
        "    </tr>\n",
        "  </thead>\n",
        "  <tbody>\n",
        "    <tr>\n",
-       "      <th>0</th>\n",
-       "      <td>output</td>\n",
-       "      <td>0</td>\n",
-       "      <td>1</td>\n",
-       "      <td>http://www.wikidata.org/prop/direct/P206</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
        "      <th>1</th>\n",
-       "      <td>output</td>\n",
-       "      <td>0</td>\n",
-       "      <td>2</td>\n",
-       "      <td>http://www.wikidata.org/prop/direct/P2044</td>\n",
+       "      <td><a target=\"_blank\" href=\"http://www.wikidata.org/entity/Q2119\">Mannheim</a></td>\n",
```

Comment:
```suggestion
       "      <td><a target=\"_blank\" href=\"https://www.wikidata.org/entity/Q2119\">Mannheim</a></td>\n",
```

Reply:
No, that is an output cell conform with the URIs from Wikidata as asked in the challenge.